doc/chips/9.rst

```diff
@@ -21,22 +25,22 @@ Chapel's current approach of bundling any extra modules in the standard library
 isn't sustainable if we wish to have a healthy ecosystem of libraries. There
 are several problems with the current strategy:
 
-  * Modules are in our repository, so:
+* Modules are in our repository, so:
 
-    * Developers must sign a CLA
-    * Code must be under a compatible license
-    * The core team needs to review each module
+  * Developers must sign a CLA
+  * Code must be under a compatible license
+  * The core team needs to review each module
 
-  * Modules are gated for release alongside the compiler
+* Modules are gated for release alongside the compiler
 
 Description
 -----------
 
 There are a several components of this package manager here I will propose what
 two of them will look like initially:
 
-  * mason - a command line tool for building Chapel programs
-  * mason.toml - module metadata mason uses to build your application
+* mason - a command line tool for building Chapel programs
+* mason.toml - module metadata mason uses to build your application
```

Comment:
Cargo capitalizes their toml files so that they're near Makefiles. Should we go with Mason.toml?